go/web.config

```
@@ -10,6 +10,12 @@
                   <action type="Redirect" url="https://marc.durdin.net/2018/03/the-case-for-keyman/" />
                 </rule>
 
+                <!-- Developer 10.0 onward redirects for package guide -->
+                <rule name="developer-help-mobile / developer-help-packages" stopProcessing="true">
+                    <match url="^([1-9][0-9])([0-9])/developer-help-(mobile|packages)(/)?$" />
```

```suggestion
                    <match url="^(([1-9][0-9])([.]?)([0-9]))/developer-help-(mobile|packages)(/)?$" />
```

Suggest we make the [.] an optional addition so in later versions we can use a cleaner version number insertion. Also, if we ever did a 13.1 we need to use the whole version number

---

I actually think later versions need to do `/go/developer/<version>/help-packages`

---

I can add your regex.

> I actually think later versions need to do `/go/developer/<version>/help-packages

go/developer/web.config already has
```
                <rule name="/go/developer/X.Y/help-(mobile|packages)" stopProcessing="true">
                  <match url="^([1-9][0-9])\.([0-9])/help-(mobile|packages)" />
                  <action type="Redirect" url="https://help.keyman.com/developer/{R:1}.0/guides/distribute/packages" />
                </rule>
```